client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -430,6 +430,13 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 				device: resolveDeviceTypeByViewPort(),
 			} );
 
+			if ( shouldLimitGlobalStyles ) {
```

Comment:
I think this would track more "try" events than intended. For instance if the site is on a Free plan, has the `wpcom-limit-global-stickers` sticker and select the default variation.

I think you should perform the same checks that are done here:  

https://github.com/Automattic/wp-calypso/blob/adb21abb8393c113d277bd11dd4ee644a07a9028/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx#L521-L525

Reply:
@mmtr I went with a separate function for the "Try" button.

I wasn't super happy of reusing the same `pickDesign` function with an additional condition as, while they technically do the exact same thing, they are semantically opposite, and could have been confusing for casual readers of the code. 🙂

Reply:
Nice approach!